### PR TITLE
Fixes mutant eyes

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Fixes https://github.com/tgstation/tgstation/issues/24317

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Thanks @XDTM

**Comment by Quardbreak** [comment]
Hmm.. What about a tongue?

**Comment by XDTM** [comment]
Pretty sure tongues work properly

**Comment by coiax** [comment]
Tongues are always mutant organs, baseline humans just have a regular tongue in their mutant organs list. We PROBABLY need a better way of specifying a subspecies organ layout.

**Comment by XDTM** [comment]
maybe a ("slot" = organtype) list?

**Comment by coiax** [comment]
Doesn't inherit very well. We could have vars for every "generic" organ type, and then have an `extra_organs` list for extra ones.

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -122,6 +123,13 @@
 		lungs.Remove(C)
 		qdel(lungs)
 		lungs = null
+
+	if(eyes)
+		eyes.Remove(C)
+		qdel(eyes)
```
Comment (AnturK) [inline code/modules/mob/living/carbon/human/species.dm]: It does, remove redundancy please Kor.

**Pull request merged**